src/web/components/ParticipantManagement/ApprovedParticipantItem.tsx

```diff
@@ -33,6 +42,18 @@ export function ApprovedParticipantItem({ participant }: ApprovedParticipantProp
           showRoleTooltip
         />
       </td>
+      <td>
+        <UpdateParticipantDialog
+          apiRoles={apiRoles}
+          onUpdateParticipant={onUpdateParticipant}
+          participant={participant}
+          triggerButton={
+            <button type='button' className='transparent-button'>
+              Edit
```

Comment:
Let's use 
```
<FontAwesomeIcon icon='pencil' />
```
instead of `Edit`

Reply:
Sounds good!
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/152250444/5a156672-602e-4b9d-83d8-c36773d659fa)